### ISSUE TITLE
[Question] Anyone Tried to Print Prusa Parts in PC or any High Temp Filament?

### ISSUE THREAD

**Issue by lew-robobionics** [prusa3d/Prusa-i3-Plus#142]
Did anyone try to Print the Prusa MK3 or MMU2S parts in PC or any High Temp Filament to continue using the Prusa inside an enclosure?

Which filament and what were the results?

**Comment by livingstonjjks** [comment]
Yes. I have printed the following as these parts have sagged or melted in Prusament PC blend:
Fan shroud
Extruder body
Extruder cover
Extruder idler mmu2s (This solved the filament loading unreliability issue I had)
